src/styles/base/_typography.scss

```diff
@@ -21,7 +21,9 @@ html {
 
 body {
   @include font-styles(body);
-  background-color: var(--color-text-background);
+  @include inTargetWeb {
```

Comment:
You might consider removing this change and simply setting the `--color-text-background` to `transparent` in the place where that variable is being overwritten for IDE targets.